.editorconfig

```diff
@@ -6,7 +6,7 @@ root = true
 
 # Change these settings to your own preference
 indent_style = tab
-indent_size = 2
+indent_size = 4
```

Comment:
I think we should just remove this, everyone can choose their own tabsize, it doesn't change the actual characters in the file.